assets/js/components/globalfilter.vue

```diff
@@ -0,0 +1,135 @@
+<template>
```

Comment:
this has some... issues, but we'll fix those in a separate branch. merge it as it is for now.

(please don't mix features in branches...)